packages/catalog-platformstate-writer/src/index.ts

```diff
@@ -0,0 +1,40 @@
+import { EachMessagePayload } from "kafkajs";
+import { logger, decodeKafkaMessage } from "pagopa-interop-commons";
+import { runConsumer } from "kafka-iam-auth";
+import { EServiceEvent } from "pagopa-interop-models";
+import { match } from "ts-pattern";
+import { DynamoDBClient } from "@aws-sdk/client-dynamodb";
+import { handleMessageV1 } from "./consumerServiceV1.js";
+import { handleMessageV2 } from "./consumerServiceV2.js";
+import { config } from "./config/config.js";
+
+const dynamoDBClient = new DynamoDBClient({
+  credentials: { accessKeyId: "key", secretAccessKey: "secret" },
+  region: "eu-central-1",
```

Comment:
Aws region hardcoded? Is this done on purpose?

Reply:
Fixed